src/mem_usage.txt

```diff
@@ -0,0 +1,39 @@
+Release/RF24Log.elf:     file format elf32-avr
+000004c2 l     F .text	00000022 _ZN14RF24LogHandler3logEhPK19__FlashStringHelperS2_PPv
+000004e4 l     F .text	00000022 _ZN14RF24LogHandler3logEhPK19__FlashStringHelperPKcPPv
+00000506 l     F .text	00000046 _ZN10RF24Logger4infoIPK19__FlashStringHelperEEvS3_T_z.constprop.15
+0080044b l     O .bss	00000002 rf24Logger
+0000054c l     F .text	00000046 _ZN10RF24Logger5errorIPKcEEvPK19__FlashStringHelperT_z.constprop.14
+00000592 l     F .text	00000046 _ZN10RF24Logger4warnIPKcEEvPK19__FlashStringHelperT_z.constprop.13
+000005d8 l     F .text	00000046 _ZN10RF24Logger4infoIPKcEEvPK19__FlashStringHelperT_z.constprop.12
+0000061e l     F .text	00000046 _ZN10RF24Logger5debugIPKcEEvPK19__FlashStringHelperT_z.constprop.11
+00000664 l     F .text	00000046 _ZN10RF24Logger5traceIPKcEEvPK19__FlashStringHelperT_z.constprop.10
+000006aa l     F .text	00000046 _ZN10RF24Logger5errorIPK19__FlashStringHelperEEvS3_T_z.constprop.9
+000006f0 l     F .text	00000046 _ZN10RF24Logger4warnIPK19__FlashStringHelperEEvS3_T_z.constprop.8
+00000736 l     F .text	00000046 _ZN10RF24Logger5debugIPK19__FlashStringHelperEEvS3_T_z.constprop.7
+0000077c l     F .text	00000046 _ZN10RF24Logger5traceIPK19__FlashStringHelperEEvS3_T_z.constprop.6
+000007c2 l     F .text	00000046 _ZN10RF24Logger3logIPKcEEvhPK19__FlashStringHelperT_z.constprop.5
+00000808 l     F .text	00000046 _ZN10RF24Logger3logIPK19__FlashStringHelperEEvhS3_T_z.constprop.4
+00000944 l     F .text	000000ac _ZN20RF24StreamLogHandler14appendLogLevelEh
+00800112 l     O .data	00000006 _ZL16rf24logLevelWarn
+00800100 l     O .data	00000006 _ZL17rf24logLevelTrace
+00800118 l     O .data	00000006 _ZL17rf24logLevelError
+0080010c l     O .data	00000006 _ZL16rf24logLevelInfo
+00800106 l     O .data	00000006 _ZL17rf24logLevelDebug
+000009f0 l     F .text	00000268 _ZN20RF24StreamLogHandler12appendFormatEcPPv
+00000c58 l     F .text	00000090 _ZN20RF24StreamLogHandler15appendTimestampEv
+00000ce8 l     F .text	0000009e _ZN20RF24StreamLogHandler4log0EhPK19__FlashStringHelperS2_PPv
+00000d86 l     F .text	0000008e _ZN20RF24StreamLogHandler4log0EhPK19__FlashStringHelperPKcPPv
+00000f58 l     F .text	00000080 _GLOBAL__I_65535_0_RF24DualLogHandler.cpp.o.1925
+00800446 l     O .bss	00000005 rf24SerialLogHandler
+00800130 l     O .data	00000008 _ZTV20RF24StreamLogHandler
+
+Core:
+.bss  0x002
+.data 0x000
+.text 0x38C
+
+Stream handler:
+.bss  0x005
+.data 0x026
+.text 0x4D0
```

Comment:
This is interesting. If I calculated correctly, the memory usage (for .data and .text) is a bit lower than before.